### PR TITLE
New rules for additifs-alimentaires.net

### PR DESCRIPTION
**Pull request by Harest**

### CODE REVIEW

File: src/chrome/content/rules/additifs-alimentaires.net.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<ruleset name="additifs-alimentaires.net">
+	<target host="additifs-alimentaires.net" />
+	<target host="www.additifs-alimentaires.net" />
```
Comment (pipboy96) [inline src/chrome/content/rules/additifs-alimentaires.net.xml]: `<target host="joomla.additifs-alimentaires.net" />`

File: src/chrome/content/rules/additifs-alimentaires.net.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (pipboy96) [inline src/chrome/content/rules/additifs-alimentaires.net.xml]: `<securecookie host=".+" name=".+" />`

File: src/chrome/content/rules/additifs-alimentaires.net.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+	<rule from="^http:"
+		to="https:" />
```
Comment (pipboy96) [inline src/chrome/content/rules/additifs-alimentaires.net.xml]: Keep this on one line, like this: `<rule from="^http:" to="https:" />`.

Comment (Harest) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: It's from the first copy/pasta of [the contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#example-ruleset-after-style-guidelines-are-applied-with-test-urls) (example ... after ... section). I'll change that.

File: src/chrome/content/rules/additifs-alimentaires.net.xml (hunk 2 above)
Comment (Harest) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: Cf. contributor guide : `For securecookie tags, if you know that all cookies on the included targets can be secured (which in particular means that the cookies are not used by any of its non-securable subdomains)`. How do you know it's the case ? Or is it now enforced for every new ruleset ?

Comment (pipboy96) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: @Harest Generally, if `<securecookie>` doesn't cause any issues, it should be added.

Comment (Harest) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: Okay, i'll keep that in mind and add it then. Thanks for the info.

File: src/chrome/content/rules/additifs-alimentaires.net.xml (hunk 1 above)
Comment (Bisaloo) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: I'd actually prefer we don't add internal subdomains. We've had complaints from domain owners about this in multiple instances in the past.

There is almost no chance a user will visit it anyways so there is not real risk to drop it IMO.

@Harest, could you revert that please?

**Comment by Bisaloo** [comment]
For future reference, there is active mixed content triggering MCB at https://www.additifs-alimentaires.net/livres.php but it doesn't look like switching to HTTPS breaks anything important.

**Comment by pipboy96** [comment]
@Bisaloo IMO if only thing broken by upgrade to HTTPS is trackers/ads, it's fine.

File: src/chrome/content/rules/additifs-alimentaires.net.xml (hunk 1 above)
Comment (Harest) [reply src/chrome/content/rules/additifs-alimentaires.net.xml]: Damn i just saw that by luck @Bisaloo, i had no notification. I'll remove it right now.

**Comment by Bisaloo** [comment]
Thank you!

**Pull request merged**